product-mini/platforms/esp-idf/CMakeLists.txt

```diff
@@ -3,53 +3,57 @@
 
 # from ESP-IDF 4.0 examples/build_system/cmake/idf_as_lib
 cmake_minimum_required(VERSION 3.5)
+project(wamr_on_esp32c3)
 
-project(wamr_esp_idf C)
+enable_language(ASM)
 
-enable_language (ASM)
+set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -O3")
```

Comment:
Had better use
```cmake
if (NOT CMAKE_BUILD_TYPE)
  set(CMAKE_BUILD_TYPE Release)
endif ()
```
So by default it is `-O3` and developer can use `cmake -DCMAKE_BUILD_TYPE=Debug/Minimal` to set it to others.